apps/balancer-tools/src/lib/trimTrailingValues.test.ts

```diff
@@ -0,0 +1,71 @@
+import { trimTrailingValues } from "./utils";
```

Comment:
isn't this just a filter? 👀 do we expect to change it's functionality and want that change to propagate everywhere?